.github/workflows/build-and-test.yml

```diff
@@ -62,7 +62,7 @@ jobs:
       - name: "Set up PostgreSQL"
         uses: harmon758/postgresql-action@v1
         with:
-          postgresql version: '9.4'  # minimum NAV requirement
+          postgresql version: '9.6'  # minimum NAV requirement
```

Comment:
We agreed on this, but this also requires:

1. Adding a release note in NOTES.rst
2. Changing the documented PostgreSQL requirements in the install docs (grepping for `9.4` will probably do)